src/core/types.h

```diff
@@ -291,6 +293,7 @@ class SourceInfo {
     CFG* cfg;
     bool is_generator;
     std::string fn; // equivalent of code.co_filename
+    std::shared_ptr<LivenessAnalysis> liveness_info;
```

Comment:
Hmm I do remember this being a memory hit, but I think it's ok for now.  I think we have some options going forward for how to potentially not need it, such as not going through the hashtable for the baseline jit, or even just generating the relevant liveness info when we generate the cfg.  or eventually doing the work of migrating to a real bytecode format :P